src/handlers.pxi

```diff
@@ -244,6 +259,9 @@ async def fuse_mkdir_async (_Container c, name):
             c.parent, name, c.mode, ctx)
     except FUSEError as e:
         ret = fuse_reply_err(c.req, e.errno)
+    except BaseException as e:
```

Comment:
Could this be written as just `except:`?

Reply:
Probably, yes. The thing is, I tend to use an explicit `BaseException` to clarify that I have thought about this and didn't just blindly capture everything. Too many bare `except:` clauses out there should really be capturing at most `except Exception:`.